components/event-calendar.vue

```diff
@@ -132,13 +91,30 @@ export default {
     }
   },
   methods: {
-    eventGroup(event) {
-      return groupForEvent(event, this.$store.state.groups.all)
-    },
     eventsOnDay(day) {
-      return this.events.filter(event =>
+      // get the events on this day
+      var events = this.events.filter(event =>
         isSameDay(new Date(event.startTime), day)
       )
+
+      var results = events.reduce(function(acc, obj) {
+        var key = obj['group']
```

Comment:
Do you mind replacing `var` with `const` in these three places? It will help enforce better scoping and let the browser optimize memory use a little better.

Reply:
Not a problem